Gemfile

```diff
@@ -14,7 +16,16 @@ group :development, :test do
   gem "jeweler", "~> 1.8.4"
   gem 'pry'
   gem 'pry-nav'
-  gem 'pry-stack_explorer'
+  gem 'pry-stack_explorer', platform: :ruby
   gem 'webmock'
   gem 'sinatra'
 end
+
+@@check ||= at_exit do
+  begin
+    require 'lock_jar/bundler'
```

Comment:
It will, but it is wrapped in `rescue LoadError` to silently fail when it cannot find the `lock_jar` gem in the Bundle.

This block is a helper to regenerate the Jarfile.lock when bundler is called, removing the additional step of having to lock it separately.